src/main/java/racing/Rule.java

```diff
@@ -0,0 +1,71 @@
+package racing;
+
+import java.util.*;
+
+public class Rule {
+    boolean isGame = true;
+    int round = 0;
+    String[] winnerRacer;
+    int winnerSpeed = 0;
+    HashMap<String, Integer> finalWinner;
+
+    void setWinnerList() {
+        finalWinner = new LinkedHashMap<String, Integer>();
+    }
+
+    void setWinnerCount(String name, int count) {
+        finalWinner.put(name, count);
+    }
+
+
+    void NameLengthCheck(String name) {
+        if (name.length() > 4) {
+            isGame = false;
+            System.out.println(" 5 < name length ");
+        }
+    }
+
+    void setRound(int round) {
+        this.round = round;
+    }
+
+
+    void finalWinner() {
```

Comment:
```suggestion
    String finalWinner() {
```
예를들어 `finalWinner` 의 메서드인 경우 void 타입보단 `finalWinner`가 누구인지 알려주면 `finalWinner` 객체가 최종 승리자를 알려주는 메서드구나 라고 인지 할 수 있을 것 같아요!!